olive/passes/onnx/genai_model_exporter.py

```diff
@@ -98,16 +122,56 @@ def _run_for_config(
             # snapshots that can be reused for future runs
             cache_dir = str(Path(tempfile.gettempdir()) / "hf_cache")
 
-        # currently we only support regular hf models so we can pass the name_or_path directly to model_name
-        # could also check if it is a locally saved model and pass the path to input_path but it is not necessary
+        extra_args = {"filename": model.onnx_file_name}
```

Comment:
I think the filename should be `str(output_model_filepath.name)` like before for `metadata_only=False`?

Reply:
actually after line 102, just using str(output_model_file.name) would work for both.

Reply:
Done!